src/less/button/ds4/button-sizes.less

```diff
@@ -1,40 +1,70 @@
-button.btn--small,
-button.expand-btn--small,
-a.fake-btn--small {
-    font-size: @ds4-font-size-12;
-    padding: 0 32px;
-}
+button.btn,
+button.expand-btn,
+a.fake-btn,
+a.cta-btn {
+    &[class*="-fixed"] {
+        [class*="__icon"] {
+            overflow: visible;
+        }
 
-button.btn--medium,
-button.expand-btn--medium,
-a.fake-btn--medium {
-    font-size: @ds4-font-size-14;
-    padding: 0 48px;
-}
+        svg[class*="icon"] {
+            align-self: center;
+            height: 1rem;
+            width: 1rem;
+        }
 
-button.btn--large,
-button.expand-btn--large,
-a.fake-btn--large {
-    font-size: @ds4-font-size-16;
-    padding: 0 80px;
-}
+        [class*="__cell"] [class*="__icon"] {
+            align-self: center;
+        }
+    }
+
+    &[class*="--small"] {
```

Comment:
Is there a way to use a mixin for this and the following ones?
It seems like a lot of repeated code, maybe can write a mixin and take the given variables and either calculate them or pass them in.

Reply:
I'll look at that.